modules/prebid/ortb2blocking/hook_bidderrequest.go

```diff
@@ -16,11 +16,11 @@ func handleBidderRequestHook(
 	cfg config,
 	payload hookstage.BidderRequestPayload,
 ) (result hookstage.HookResult[hookstage.BidderRequestPayload], err error) {
-	if payload.BidRequest == nil {
-		return result, hookexecution.NewFailure("empty BidRequest provided")
+	if payload.BidRequest == nil || payload.BidRequest.BidRequest == nil {
+		return result, hookexecution.NewFailure("empty input provided")
```

Comment:
How about we use the same error message we now use in the mutation code: `"payload contains a nil bid request`"?